docs/guides/audio-config.md

```diff
@@ -57,9 +57,9 @@ does not offer a great deal of flexibility, it might work for getting acceptable
 
 All of apple's fancy tuning of the speakers is done in macOS, but a similar configuration is currently available for only the MacBook Pro 16 inch 2019.
 
-### Macbook pro 16,1 2019
+### Macbook pro 16 2019
 
-Currently we have an experimental DSP(Digital Signal Processing) config for Macbook Pro 16,1 2019 with 6 speakers.
+Currently we have an experimental DSP(Digital Signal Processing) config for Macbook Pro 16,1 and 16,4 with 6 speakers.
```

Comment:
I believe users will be more comfortable with MacBook Pro 16 inch, 2019 rather than 16,1/16,4